docs/account.md

```diff
@@ -108,45 +108,45 @@ Make sure the PHP GMP extension is loaded! Uncomment `extension=gmp` in your php
 
 [VerifySRP6Login.php](https://gist.github.com/Treeston/34d9249fb467dddc11b2568e74f8cb1e)
 
-### session_key
+### session\_key
```

Comment:
why is this better? just wondering

Reply:
It doesnt matter. Before with first markdown we used we had to do it to escape underline.

Now with the one we use this isnt needed, but doesnt matter if it is added or not

Reply:
This is the pull list, now you can give it merge